### DISCUSSION TITLE
Welcome to Bedrock Tweaks Discussions!

### DISCUSSION THREAD

**Discussion by drav0011** [BedrockTweaks/Files#74] [Announcements]
Feel free to suggest here new features to add to the website or new packs.

For now we do not want to strive too far away from Vanilla Tweaks, but in the future might add new packs!

**Comment by FemCaramella** [comment]
I think that it would be awesome if colored and pride hearts like the ones on vanilla tweaks could be added to bedrock tweaks too

**Reply by drav0011** [reply]
Those are confirmed!
https://github.com/BedrockTweaks/Files/issues?q=is%3Aissue+is%3Aopen+hearts

**Reply by FemCaramella** [reply]
Oh! Thanks for the help!

**Comment by Ethan-qUeStIoN-mArK-lmao** [comment]
Hi! I was wondering if you could implement some texture packs that could change the enchantment glint colour (e.g gold, blue...). Thank you :)

**Reply by Ethan-qUeStIoN-mArK-lmao** [reply]
also, WHENEVER you can get to it, could you create a texture pack to make lava clear and ancient debris stand out (I've seen it recoloured to blue) thx again!!!!!!

**Reply by SquatchHunter** [reply]
Enchantment glint was discussed and is only partially possible, clear lava to my knowledge was never on vanilla tweaks, recoloured Netherite we can probably do since iirc it was on there

**Comment by Smileyr979** [comment]
Hi, I loaded up my world after creating a custom resource pack. As I looked around and all of the cobblestone slabs and walls had the untextered black and purple. Would be great to have this fixed soon👍

**Reply by Smileyr979** [reply]
I’m guessing this is due to the variated cobble option for RPs but honestly I’m not too sure

**Reply by SquatchHunter** [reply]
normally those checkerboxes only appear when you have incorrectly installed a pack

**Comment by Smileyr979** [comment]
Do you know how I would fix that on an Xbox 1 console

**Comment by WorriedPearl** [comment]
Glowing mob eyes ( and warden horns n stuff like hostile mobs only in white would be nice )

**Reply by SquatchHunter** [reply]
not really in scope, but we can take a look post addon release (which is when we are gonna start looking at community ideas for packs)

**Comment by MrSoldaAKAmayur** [comment]
https://mcpedl.com/shhmap/
I made this simple Resource pack
Ignore the 1st image but the 2nd could definitely be added 
I am sure that many players would like it 
If you want then I could modify it for Bedrock tweaks

**Reply by SquatchHunter** [reply]
interesting idea for a pack, later down the line we can consider something similar but for right now it wont be a focus

**Reply by MrSoldaAKAmayur** [reply]
Yes 
We want make bedrock tweak as good and similar as vanilla tweaks right, as you said 
I will search the vanilla tweaks to see if there's anything I could do
FOR THE MINECRAFTers!

**Reply by MrSoldaAKAmayur** [reply]
Could I create like transparent GUI
Like the vanilla tweaks 
Or even multiple one's like 0%,25%,50%,69%,75%,90%(inner part)and 0%,25%,50%,69%,75%,90%(outline)
Many ideas have popped up in my mind I would like to make if you agree 
I will be waiting for your reply

**Comment by user3685821** [comment]
Make an addon to let us place blocks on a block that is occupied by a flower, I hate having to break a flower on every block to build

**Reply by SquatchHunter** [reply]
not sure if that would be possible unforunately, thats a native game mechanic and we sadly cannot overwrite them at the moment

**Comment by WHAMZ-Tastic** [comment]
Replace the Totem of undying with a piece of bread (obviously in the fun category)

**Reply by SquatchHunter** [reply]
give me a reason _why_

**Comment by TrunksDGoat** [comment]
Add an option to change all grass to snowy grass. I think this would be cool to use during Christmas time, and you could also add some other Christmas themed textures too, but I don't have any other ideas right now. Maybe Christmas themed mobs, like they all wear santa hats or something?

**Reply by SquatchHunter** [reply]
adding santa hats might be _rather difficult_ but not impossible, just a lot of geometry needs to be modified
a santa hat item (and maybe an easter egg version thats OP) could be on the cards and it shouldnt be hard to add that to mobs that already support helmets (zombies & skeletons)

something to look into though :)

**Comment by JBHL1212** [comment]
Is it possible to add noteblock displays? Internet hasn't been very useful in finding anything, and I see that it's already a feature on vanilla tweaks, has it been too complicated to implement into bedrock tweaks or has it just been looked over lol?

**Reply by SquatchHunter** [reply]
as far as I know; No. Atleast not the way VT does it which would be ideal.
The only _known_ way to do it is via coloured and or custom particles
This mostly stems from our inability right now to modify vanilla blocks (we would have to make a custom noteblock block that would have the texture)

**Reply by JBHL1212** [reply]
What about making the colored notes that come out from the noteblock every time you hit it not just be a note particle and instead say what note you are tuning it to? Instead of the red note particle that comes out from the noteblock a red C particle comes out instead? Don't need the insturment part of noteblock displays, Wiki is helpful with that, not the notes lol

**Reply by SquatchHunter** [reply]
We have considered this before and I am not sure if we agreed that it would meaningfully solve the problem, since it means potentially have to retune everytime if you hit the wrong button verse having a clear and concise way to check the pitch of a block.
We can however revisit it.
